Idea.js

```diff
@@ -0,0 +1,21 @@
+
+class Idea {
+  constructor(title, body) {
+    this.title = title; //string
+    this.body = body; //string
+    this.starred = false;
```

Comment:
We might need an "or" and another parameter here? 
this.starred = starred || false;